documentation/src/docs/pravega-concepts.md

```diff
@@ -11,362 +11,250 @@ You may obtain a copy of the License at
 
 
-We mentioned previously that an Event is written into one of the Stream's
-Segments.  Taking into account AutoScaling, you should think of Stream Segments
-as a bucketing of Events based on Routing Key and time.  At any given time,
-Events published to a Stream within a given value of Routing Key will all appear
-in the same Stream Segment.
+As it was mentioned in the earlier part of the section, that an Event is written into one of the Stream Segments. By considering auto scaling, Stream Segments performs bucketing of Events based on Routing Key and time. It is obvious that, at any given time, Events published to a Stream with a given value of Routing Key will appear in the same Stream Segment.
 
 ![Stream Segment](img/rk.segment.new.png) 
 
-It is also worth emphasizing that Events are written into only the active Stream
-Segments.  Segments that are sealed do not accept writes.  In the figure above,
-at time "now", only Stream Segments 3, 6 and 4 are active and between those
-three Stream Segments the entire key space is covered.  
+It is also worth emphasizing that Events are written only on the active Stream
+Segments. Segments that are sealed do not accept writes. In the figure above,
+at time **now**, only Stream **Segments 3**, **6** and **4** are active and the entire key space is covered between those three Stream Segments.  
 
-### Stream Segments and ReaderGroups
+### Stream Segments and Reader Groups
 
-Stream Segments are important to understanding the way Reader Groups work.
+Stream Segments play a major role in understanding the way Reader Groups work.
 
 ![Stream Segment](img/segment.readergroup.png) 
 
-Pravega assigns each Reader in a ReaderGroup zero or more Stream Segments to
-read from.  Pravega tries to balance out the number of Stream Segments each
-Reader is assigned.  In the figure above, Reader B1 reads from 2 Stream Segments
-while each of the other Readers in the Reader Group have only 1 Stream Segment
-to read from.  Pravega makes sure that each Stream Segment is read by exactly
-one Reader in any ReaderGroup configured to read from that Stream. As Readers
-are added to the ReaderGroup, or Readers crash and are removed from the
-ReaderGroup, Pravega reassigns Stream Segments so that Stream Segments are
-balanced amongst the Readers.
+Pravega assigns _zero_ or more Stream Segments to each Reader in a Reader Group. Pravega tries to balances the number of Stream Segments assigned to each Reader. In the figure above, **Reader B1** reads from two Stream Segments (**Segment 0** and **Segment 3**), while the other Reader Group (**Reader B2**, **Reader B3**) have only only one Stream Segment to read from. Pravega makes sure that each Stream Segment is read exactly by one Reader in any Reader Group configured with that Stream. Irrespective of  Readers being added to the Reader Group or removed from the Reader Group due to crash, Pravega reassigns Stream Segments to maintain balance amongst the Readers.
```

Comment:
Do not emphasize only _zero_, but the whole definition _zero or more_.

Reply:
done